### PR TITLE
Icon resize

### PR DESCRIPTION
**Pull request by devincloudkelly**
Loaded in new svg icons and re-sized them accordingly. Replaced any references to old icons with current set.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -23,4 +23,5 @@
 	],
 	"editor.insertSpaces": false,
 	"editor.detectIndentation": false,
+	"git.ignoreLimitWarning": true,
```
Comment (jacksonrya) [inline .vscode/settings.json]: This should be in your global vscode settings. This file is what broke my git when I was trying to checkout your branch :man_shrugging:

File: app.json [hunk 2]
```diff
@@ -27,7 +27,7 @@
       "supportsTablet": true
 		},
 		"extra": {
-			"variant": "donor"
+			"variant": "client"
```
Comment (jacksonrya) [inline app.json]: ideally, you exclude changes like these from your commits

File: .vscode/settings.json (hunk 1 above)
Comment (devincloudkelly) [reply .vscode/settings.json]: Removed and updated locally. Thanks!

File: app.json (hunk 2 above)
Comment (devincloudkelly) [reply app.json]: Reverted this to original.

**Comment by jacksonrya** [comment]
Rebased March 31, 13:11. History cleaned up, ready to squash n merge

**Comment by devincloudkelly** [comment]
@jacksonrya Thanks for the update - I'm having some local environment issues with rebasing. Can't rebase -i without hitting an error and haven't resolved yet, so haven't been able to squash all my commits. Working on that now, then I'll merge this.

**Comment by jacksonrya** [comment]
I received a message about issues with rebasing or squashing locally? You can squash on github! Just hit the arrow next to 'Create a new commit' button

**Comment by jacksonrya** [comment]
**Update**
- Sangkee added two more icons to Figma: an open and closed eye

**Pull request merged**